hydrolib/core/io/bc/models.py

```diff
@@ -236,6 +291,152 @@ class T3D(ForcingBase):
     _verticalpositiontype_validator = get_enum_validator(
         "verticalpositiontype", enum=VerticalPositionType
     )
+    _timeinterpolation_validator = get_enum_validator(
+        "timeinterpolation", enum=TimeInterpolation
+    )
+
+    @root_validator(pre=True)
+    def _validate_quantityunitpair(cls, values):
+        super()._validate_quantityunitpair(values)
+
+        quantityunitpairkey = "quantityunitpair"
```

Comment:
this var is only used once, on the next line. Maybe just have the "quantityunitpair" directly in that dict-lookup and remove this variable?

Reply:
Good point! I stole this from the base implementation, where it is in fact used more often. Since it is only used once here, I will remove this line.

Reply:
Done